contracts/staking/contracts/src/StakingProxy.sol

```diff
@@ -42,7 +42,15 @@ contract StakingProxy is
         MixinStorage()
     {
         readOnlyProxy = _readOnlyProxy;
+
+        // Deployer address must be authorized in order to call `init`
+        _addAuthorizedAddress(msg.sender);
```

Comment:
I'm actually okay with this pattern, but it does seem like it would be more ergonomic to just prevent `init()` from being called when params are set than to use `onlyAuthorized` here. This is definitely better than just authorizing the owner by default though.

Reply:
Discussed offline, and we are keeping this pattern the same.